### ISSUE TITLE
Can't get the background transparency to work

### ISSUE THREAD

**Issue by yannbcf** [Shopify/screenshot-glb#39]
Hello, i don't know if it's a model-viewer issue, but the help info of screenshot-glb says that by default, the background color is transparent.

For me it's a pure ``#FFFFFF`` white as background color, my image format is ``png``

This is the npm script i use:
```
"image": "screenshot-glb -i inputPath -o outputPath -w 512 -h 512"
```

If you need any other information let me know

**Comment by abdulwahid001** [comment]
Hello did you get it to work ?

**Comment by YannBCF** [comment]
No, even by modifying to html background of the model viewer instance

**Comment by MattSmillie** [comment]
See the issue I posted here:
https://github.com/Shopify/screenshot-glb/issues/34

There is a bug which results in only jpegs getting output. In the issue I linked you can see a tweak which you can make to get png to output.
